### ISSUE TITLE
Información Keras metrics, loss function, optimicer...

### ISSUE THREAD

**Issue by josprimen** [josprimen/TFG#6]
Search information about optimicer, loss function, metrics, activation...in keras

**Comment by josprimen** [comment]
Links:
- https://github.com/keras-team/keras/tree/master/keras (Github de Keras con el código de métricas, funciones de pérdida...)

**Comment by josprimen** [comment]
Hemos buscado un segundo ejemplo guiado para realizar, es sobre clasificación de películas sobre imdb.
Podemos encontrarlo en el segundo punto en el github de deep learning de F.Chollet.

Descubrimos que no debe usarse el mismo conjunto de datos para entrenar una red que como conjunto de prueba.

**Comment by josprimen** [comment]
Respecto a las métricas, funciones de pérdida, etc, descubrimos que hemos de probar distintas configuraciones hasta dar con una combinación que parezca óptima para nuestro problema.

Si queremos información específica sobre alguna de estas, tenemos en el github de keras el código que las compone.
